RIGS/signals.py

```diff
@@ -102,3 +105,35 @@ def on_revision_commit(sender, instance, created, **kwargs):
 
 
 post_save.connect(on_revision_commit, sender=models.EventAuthorisation)
+
+
+def send_admin_awaiting_approval_email(user, request, **kwargs):
+    # Bit more controlled than just emailing all superusers
+    for admin in models.Profile.objects.filter(email__in=[y for x in settings.ADMINS for y in x]):
+        # Check we've ever emailed them before and if so, if cooldown has passed.
+        if admin.last_emailed is None or admin.last_emailed + datetime.timedelta(minutes=15) <= timezone.now():
+            context = {
+                'request': request,
+                'link_suffix': '/admin/RIGS/profile/?is_approved__exact=0',
+                'number_of_users': models.Profile.users_awaiting_approval_count(models.Profile.objects),
```

Comment:
This should probably be a `class_method`, should stop you having to pass it in

Reply:
Yeah I'm not sure what I did here I just poked it until it worked...

Reply:
Fair! `class_method` is the equivalent of static